Formula/lc0.rb

```diff
@@ -20,25 +20,48 @@ class Lc0 < Formula
   depends_on "python@3.9" => :build # required to compile .pb files
   depends_on "eigen"
 
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "gcc" # for C++17
+    depends_on "openblas"
+  end
+
+  fails_with gcc: "5"
+
   resource "network" do
     url "https://training.lczero.org/get_network?sha=00af53b081e80147172e6f281c01daf5ca19ada173321438914c730370aa4267", using: :nounzip
     sha256 "12df03a12919e6392f3efbe6f461fc0ff5451b4105f755503da151adc7ab6d67"
   end
 
+  # Fix compile error due to missing #include <condition_variable>
+  # Remove in the next release
+  patch do
+    url "https://github.com/LeelaChessZero/lc0/commit/997257a16fc74c848ab55e475c4d233d78838506.patch?full_index=1"
+    sha256 "5f4fb6b730a04bfd36567302bc4ad9d22f7f69c55c6fad35047a18d514210758"
+  end
+
   def install
-    system "meson", *std_meson_args, "-Dgtest=false", "build/release"
+    args = ["-Dgtest=false"]
+    on_linux do
+      args << "-Dopenblas_include=#{Formula["openblas"].opt_include}"
+      args << "-Dopenblas_libdirs=#{Formula["openblas"].opt_lib}"
+    end
+    system "meson", *std_meson_args, *args, "build/release"
 
     cd "build/release" do
       system "ninja", "-v"
       libexec.install "lc0"
     end
 
     bin.write_exec_script libexec/"lc0"
-    libexec.install resource("network")
+    resource("network").stage { libexec.install Dir["*"].first => "42850.pb.gz" }
```

Comment:
Trying to renaming to something cleaner. Can revert to hash if preferred.

Previously, the filename was a hash 
i.e. `00af53b081e80147172e6f281c01daf5ca19ada173321438914c730370aa4267`

I decided to rename based on how upstream named the Windows file: https://lczero.org/play/download/
> - **703810.pb.gz** (a default network file), and

They have a section on Homebrew too https://lczero.org/play/download/#macos
> The Homebrew package includes the **lc0** engine and network **42850**.

Reply:
I agree that this is a better name that the sha256 hash.  Nice work!